### ISSUE TITLE
Cannot find module '@expo/image-utils'

### ISSUE THREAD

**Issue by vigneshraaj** [expo/expo-cli#756]
When I starts or restarts a local server for your app, its failing to start with following error.It was working fine until I updated expo-cli to 2.20.3 and expo-sdk to 33.0.7

Expecting it to provide URL of local server.

```
$ expo start
Starting project at /Users/xyz/projects/proj-test/code
Expo DevTools is running at http://localhost:19002
Cannot find module '@expo/image-utils'
Error: Cannot find module '@expo/image-utils'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/@expo/xdl@55.0.3/src/AssetUtils.js:6:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

```
$ expo diagnostics

  Expo CLI 2.20.3 environment info:
    System:
      OS: macOS 10.14.4
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.0 - /usr/local/opt/node@10/bin/node
      Yarn: 1.12.3 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/opt/node@10/bin/npm
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5314842
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      expo: 33.0.7 => 33.0.7 
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.8 => 0.59.8 
      react-navigation: 3.11.0 => 3.11.0 
    npmGlobalPackages:
      expo-cli: 2.20.3`
```

**Comment by cruzach** [comment]
duplicate of #755

**Comment by XIAOXIAOSIYU** [comment]
The same here.

**Comment by WeZinc** [comment]
Same here

**Comment by prayas789** [comment]
i think its because of incomplete download  of new project dependencies
try to install it again(expo-cli)

**Comment by vigneshraaj** [comment]
@cruzach As per the comment on #755, I downgraded expo-cli to v2.19.5. 
expo start is working but when I launch the application on device, its throwing error.
![Screenshot_1561400651](https://user-images.githubusercontent.com/2261556/60042725-04434600-96dc-11e9-929e-4b0fd97e22e6.png)

Seems cli v2.19.5 is not compatible with sdk v33+

**Comment by pawel-stolka** [comment]
struggling the same since an hour...

**Comment by pavini** [comment]
same problem :/

**Comment by prayas789** [comment]
yes now i am facing same issue

**Comment by prayas789** [comment]
[SOLVED]
npm install -g expo-cli@2.4.3
and (rollback to 2.4.3) you are good to go

**Comment by INGJEISSON** [comment]
@prayas789  compa, y como hago el roolback?

**Comment by pawel-stolka** [comment]
YES! @prayas789 ! it works! thanls man :)

**Comment by INGJEISSON** [comment]
Yes me funcionó..

**Comment by cruzach** [comment]
Fixed in `expo-cli@2.20.5`! 🎉 

Thanks everyone for letting us know about this issue so quick, and being patient for the fix 😄
